index.html

```diff
@@ -70,9 +70,207 @@ <h1>
+        <span class="divider"></span>
+        <div class="card-wrapper">
+          <article class="card">
+            <section class="content">
+              <h4 class="title">Multi-Post Stories Gain+Glory</h4>
+              <ul>
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>Javascript</li>
+                <li>Html</li>
+              </ul>
+              <button>See Project</button>
+            </section>
+          </article>
+          <article class="card">
+            <section class="content">
+              <h4 class="title">Multi-Post Stories Gain+Glory</h4>
+              <ul>
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>Javascript</li>
+                <li>Html</li>
+              </ul>
+              <button>See Project</button>
+            </section>
+          </article>
+          <article class="card">
+            <section class="content">
+              <h4 class="title">Multi-Post Stories Gain+Glory</h4>
+              <ul>
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>Javascript</li>
+                <li>Html</li>
+              </ul>
+              <button>See Project</button>
+            </section>
+          </article>
+          <article class="card">
+            <section class="content">
+              <h4 class="title">Multi-Post Stories Gain+Glory</h4>
+              <ul>
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>Javascript</li>
+                <li>Html</li>
+              </ul>
+              <button>See Project</button>
+            </section>
+          </article>
+          <article class="card">
+            <section class="content">
+              <h4 class="title">Multi-Post Stories Gain+Glory</h4>
+              <ul>
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>Javascript</li>
+                <li>Html</li>
+              </ul>
+              <button>See Project</button>
+            </section>
+          </article>
+          <article class="card">
+            <section class="content">
+              <h4 class="title">Multi-Post Stories Gain+Glory</h4>
+              <ul>
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>Javascript</li>
+                <li>Html</li>
+              </ul>
+              <button>See Project</button>
+            </section>
+          </article>
+        </div>
+      </section>
+      <!-- about section -->
+      <section class="about-section">
+        <article class="bio-container">
+          <h1 class="heading">About me</h1>
+          <p class="desc">
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
```

Comment:
<img width="1011" alt="1" src="https://user-images.githubusercontent.com/77843050/220863667-8921cbbc-9b80-4b51-ad3c-2e88d7870942.png">

- [ ] we are missing these background images in your design, please make sure you have them and resubmit your work